Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -94,22 +95,27 @@ bash scripts/install_prerequisites.sh
 
    For making any changes to original repository, we first sync our cloned repository with original repository. We merge develop with `upstream/develop` to do this. Now we make a new branch, do the changes on the branch, push the branch to forked repository, and make a PR from Github interface. We use a different branch to make changes so that we can work on multiple issues while still having a clean version in develop branch.
 
-## Setup a virtual environment
+## 3. Setup a virtual environment
 
 For your virtual environment, we recommend you use [pyenv](https://github.com/pyenv/pyenv). Here are some instructions for doing so, but you can use another virtual environment tool if you wish:
-   > [NOTE!]
-   > The commands below can be executed in any directory, as they are not path-specific.
-1. **Make sure you install the Python build dependencies for your operating system. These are specified [here](https://github.com/pyenv/pyenv/wiki#suggested-build-environment). If you don't do this it might lead to problems further on.** The build dependencies for Ubuntu/Debian are
+
+> [NOTE!]
+> The commands below can be executed in any directory, as they are not path-specific.
+
+1. **Make sure you install the Python build dependencies for your operating system. These are specified [here](https://github.com/pyenv/pyenv/wiki#suggested-build-environment). If you don't do this it might lead to problems further on.** To install build dependencies in Ubuntu/Debian run
 
 
    ```
-   sudo apt-get install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+   sudo apt update; sudo apt install build-essential libssl-dev zlib1g-dev \
+   libbz2-dev libreadline-dev libsqlite3-dev curl git \
+   libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
    ```
 
-2. Install pyenv:
+2. Install pyenv using [installation steps](https://github.com/pyenv/pyenv?tab=readme-ov-file#installation).
```

Comment:
... using these installation steps: [link]

Or just say "Install pyenv." and link the whole text to that link.